src/utils/env/index.ts

```diff
@@ -272,10 +327,10 @@ export const getHumanReadableScopes = (scopes?: (EnvironmentVariableScope | 'pos
 export const translateFromMongoToEnvelope = (env: Record<string, string> = {}) => {
```

Comment:
```suggestion
export const translateFromMongoToEnvelope = (env: Record<string, string> = {}): EnvelopeItems[] => {
```

This reduces the need to explicitly assert strings as `const`s.

Reply:
Yeah, I had tried that but it opened up a can of worms that I decided to back out of 😞